app/src/main/java/org/tensorflow/lite/examples/classification/Presentation/CheckListActivity/CheckListActivity.java

```diff
@@ -74,5 +104,47 @@ public void onClick(DialogInterface dialogInterface, int i) {
                 }
             }
         });
+        binding.vaccineNo.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                binding.vaccine2.setVisibility(View.GONE);
+                binding.vaccine2Answer.setVisibility(View.GONE);
+                binding.vaccine3.setVisibility(View.GONE);
+                binding.first.setVisibility(View.GONE);
+                binding.second.setVisibility(View.GONE);
+            }
+        });
```

Comment:
```suggestion
        binding.vaccineNo.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                    binding.vaccinePfizer.setEnabled(false);
            }
        });
```
이런 식으로 View를 사용자가 아니오 클릭했을 때 라디오버튼을 disable하면 될 것 같아!